keps/sig-instrumentation/0034-distributed-tracing-kep.md

```diff
@@ -211,6 +230,14 @@ This documentation will put forward standards for:
 
 Having these standards in place will ensure that our tracing instrumentation works well with all backends, and that reviewers have concrete criteria to cross-check PRs against. 
 
+### Abuse potential
+
+Using objects to propagate SpanContext comes with some drawbacks.  In particular,
+
+1. Anyone with Read (get, list, watch) access to a traced object and permission to write to the trace backend can append spans to a trace.  There is not a simple way to verify that a Span was written by a particular component.
```

Comment:
First two issues sound like a trace library/backend security problem, not K8s one. 
I don't think that we have any security around instrumentation in K8s. So this seems out of scope.